templates/nginx/galaxy-dev.j2

```diff
@@ -0,0 +1,157 @@
+    }
+
+    location /favicon.ico {
+        alias {{ galaxy_server_dir }}/static/favicon.ico;
+    }
+
+    location /reports/ {
+        uwsgi_pass           127.0.0.1:9001;
+        uwsgi_param          UWSGI_SCHEME $scheme;
+        include              uwsgi_params;
+        auth_basic "Restricted Content";
+        auth_basic_user_file /etc/nginx/.htpasswd;
+    }
+
+    location /tiaas {
+        uwsgi_pass 127.0.0.1:5000;
+        uwsgi_param UWSGI_SCHEME $scheme;
+        include uwsgi_params;
+    }
+
+    location /tiaas/static {
+        alias /opt/tiaas/static;
+    }
+
+    location /join-training {
+        uwsgi_pass 127.0.0.1:5000;
+        uwsgi_param UWSGI_SCHEME $scheme;
+        include uwsgi_params;
+    }
+
+    location /api/upload/resumable_upload {
+        # Disable request and response buffering
+        proxy_request_buffering     off;
+        proxy_buffering             off;
+        proxy_http_version          1.1;
+
+        # Add X-Forwarded-* headers
+        proxy_set_header X-Forwarded-Host   $host;
+        proxy_set_header X-Forwarded-Proto  $scheme;
+
+        proxy_set_header Upgrade            $http_upgrade;
+        proxy_set_header Connection         "upgrade";
+        client_max_body_size        0;
+        proxy_pass http://localhost:{{ galaxy_tusd_port }}/files;
+    }
+
+{% if galaxy_config['galaxy']['nginx_upload_store'] is defined %}
+
+    location /_upload {
+        upload_store {{ galaxy_config['galaxy']['nginx_upload_store'] }};
+        upload_limit_rate "{{ nginx_upload_limit_rate|d(0) }}";
+        upload_store_access user:rw group:r all:r;
+        upload_pass_form_field "";
+        upload_set_form_field "__${upload_field_name}__is_composite" "true";
+        upload_set_form_field "__${upload_field_name}__keys" "name path";
+        upload_set_form_field "${upload_field_name}_name" "$upload_file_name";
+        upload_set_form_field "${upload_field_name}_path" "$upload_tmp_path";
+        upload_pass_args on;
+        upload_pass /_upload_done;
+    }
+
+    location /_upload_done {
+        set $dst /api/tools;
+        if ($args ~ nginx_redir=([^&]+)) {
+                set $dst $1;
+        }
+        rewrite "" $dst;
+    }
+
+{% endif %}
+{% if galaxy_config['galaxy']['nginx_upload_job_files_store'] is defined %}
+
+    location /_job_files {
+        if ($request_method != POST) {
+                rewrite "" /api/jobs/$arg_job_id/files last;
+        }
+        upload_store {{ galaxy_config['galaxy']['nginx_upload_job_files_store'] }};
+        upload_limit_rate "{{ nginx_upload_limit_rate|d(0) }}";
+        upload_store_access user:rw group:r all:r;
+        upload_pass_form_field "";
+        upload_set_form_field "__${upload_field_name}_path" "$upload_tmp_path";
+        upload_pass_args on;
+        upload_pass /_upload_job_files_done;
+    }
+
+    location /_upload_job_files_done {
+        internal;
+        rewrite "" /api/jobs/$arg_job_id/files;
+    }
+
+{% endif %}
```

Comment:
Can this section be removed now?
```suggestion
```

Reply:
I want to do one thing at a time. And I thni kwe are stulll using nginx uploads for job returns from pulsar atm.